### ISSUE TITLE
feature: LATEX formulas support

### ISSUE THREAD

**Issue by geekkun** [simonhaenisch/md-to-pdf#82]
**Problem:**

Right now LATEX formulas are not supported and all $$xyz$$ left as is in the output pdf.

**Solution:**

A MathJax library could be added to parse and display LATEX formulas.

**Comment by simonhaenisch** [comment]
Thanks for the feature request. Using MathJax is actually already possible.

Just add the scripts in your markdown:

```html
<script>
MathJax = {
	tex: {
		inlineMath: [['$', '$'], ['\\(', '\\)']],
	},
};
</script>
<script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"></script>

# MathJax Test

When $a \ne 0$, there are two solutions to \\(ax^2 + bx + c = 0\\).
```

Result:

<img src="https://user-images.githubusercontent.com/3410759/106542166-92f88580-6503-11eb-9df3-337ec2bde358.png" width="360">

---

I'll release a new version soon that will also allow to specify scripts in the config (which will use `page.addScriptTag` under the hood).

**Comment by simonhaenisch** [comment]
Just released 3.3.0 which has a new `script` config option, and for reference the above example is a test case in `src/test/mathjax`.
